package.json

```diff
@@ -37,7 +37,7 @@
   ],
   "bowerJSON":{
     "main": [
-      "chosen.jquery.min.js",
+      "chosen.jquery.js",
       "chosen.css",
```

Comment:
No need to change the css since it was already referencing the full version (even though we were packaging the minified css file).